mmv1/products/vertexai/terraform.yaml

```diff
@@ -13,6 +13,19 @@
 
 --- !ruby/object:Provider::Terraform::Config
 overrides: !ruby/object:Overrides::ResourceOverrides
+  Tensorboard: !ruby/object:Overrides::Terraform::ResourceOverride
+    exclude_import: true
```

Comment:
Why does the import need to be excluded?

Reply:
It wasn't working as the {{name}} extracted from the `terraform import` was passed into the read call (having URL `{{VertexAIBasePath}}{{name}}`). Actually, the name is output-only field having the format `projects/{project}/locations/{location}/tensorboards/{tensorboard}` as per the [API Docs](https://cloud.google.com/vertex-ai/docs/reference/rest/v1/projects.locations.tensorboards). I've also tried to change import_format and id_format in the terraform.yaml but that didn't work. Hence, I have to write custom_import for this to work.

Reply:
I have also added a `full` as well as an `update` test for this resource.